tests/integration/test_tenants.py

```diff
@@ -30,12 +31,30 @@ def test_tenants_POST(client, auth_headers):
         "staffIDs": [1, 2]
     }
 
+    newTenantWithLease = {
+        "firstName": "Finn",
+        "lastName": "The Human",
+        "phone": "123-555-4321",
+        "propertyID": 2,
+        "occupants": "3",
+        "dateTimeEnd": "2021-04-29T07:00:00.000Z",
+        "dateTimeStart": "2020-11-26T08:00:00.000Z",
```

Comment:
Not critical, However we should probably use dynamic dates here. If we ever add validation checks that don't allow leases to be created with dateTimeEnd and start dates that are in the past then this test will start to fail. I'm sure Python has a method that returns this time format as it is a standard format, but I'm not 100% sure.